.gitlab-ci.yml

```diff
@@ -314,7 +331,20 @@ build:base+32bit:
     variables:
       - $FULL_CI == "true"
 
-build:edge+flambda:
+build:native-ondemand:
```

Comment:
This should be named `build:base:native-ondemand`

By the way, why is this build needed? Sorry if I missed that part.

Reply:
For engine_bench that uses native_compute (`-native-compiler no` would fallback to vm_compute which is too slow) but for which native_compiler breaks (out of memory). It thus requires `-native-compiler ondemand`